polynote-frontend/polynote/data/result.ts

```diff
@@ -251,9 +251,9 @@ export class ResultValue extends Result {
                 const el = div([], [
                     h4(['result-name-and-type'], [
                         span(['result-name'], [this.name]), ': ', resultType,
-                        iconButton(['view-data'], 'View data', '', '[View]')
+                        iconButton(['view-data'], 'View data', 'table', '[View]')
                             .click(_ => valueInspector.inspect(this, cell.path, 'View data')),
-                        iconButton(['plot-data'], 'Plot data', '', '[Plot]')
+                        iconButton(['plot-data'], 'Plot data', 'chart-bar', '[Plot]')
```

Comment:
Must have been fun to track down what icon name each random character represented 😅